src/wp-includes/option.php

```diff
@@ -2144,20 +2144,47 @@ function update_network_option( $network_id, $option, $value ) {
 	 */
 	$value = apply_filters( "pre_update_site_option_{$option}", $value, $old_value, $option, $network_id );
 
+	$is_multisite = is_multisite();
```

Comment:
Nit: `is_multisite()` is a very fast function. You can probably call it directly below in place of storing the value in a separate variable. Otherwise, I'd at least move this variable down to just before the code block where it's first used.

Reply:
+1, no need for this change, I would suggest inlining the `is_multisite()` check, it also avoids making an unrelated change to existing code.

Reply:
Thanks for the feedback. Addressed in https://github.com/WordPress/wordpress-develop/pull/5434/commits/5b42a03b0c539b4ada1cb51bbda75a373b044d33